src/applications/vaos/appointment-list/components/CommunityCareAppointmentDetailsPage.jsx

```diff
@@ -0,0 +1,168 @@
+
+import moment from '../../lib/moment-tz';
+
+import { FETCH_STATUS } from '../../utils/constants';
+import { scrollAndFocus } from '../../utils/scrollAndFocus';
+import * as actions from '../redux/actions';
+import AppointmentDateTime from './cards/confirmed/AppointmentDateTime';
+import { getVARFacilityId } from '../../services/appointment';
+import AppointmentInstructions from './cards/confirmed/AppointmentInstructions';
+import AddToCalendar from '../../components/AddToCalendar';
+import { selectFeatureCancel } from '../../redux/selectors';
+import FacilityAddress from '../../components/FacilityAddress';
+
+function CommunityCareAppointmentDetailsPage({
+  appointmentDetails,
+  appointmentDetailsStatus,
+  fetchConfirmedAppointmentDetails,
+  confirmedStatus,
+}) {
+  const { id } = useParams();
+  const history = useHistory();
+
+  const appointment = appointmentDetails?.[id];
+
+  useEffect(() => {
+    const status = confirmedStatus === FETCH_STATUS.succeeded;
+
+    if (!status) {
+      history.push('/');
+    }
+
+    if (!appointment) {
+      fetchConfirmedAppointmentDetails(id);
+    }
+
+    scrollAndFocus();
+  }, []);
+
+  if (appointmentDetailsStatus === FETCH_STATUS.loading) {
+    return (
+      <div className="vads-u-margin-y--8">
+        <LoadingIndicator message="Loading your confirmed appointments..." />
+      </div>
+    );
+  }
+
+  if (!appointment) {
+    return null;
+  }
+
+  const header = 'Community Care';
+  const location = appointment.contained.find(
+    res => res.resourceType === 'Location',
+  );
+
+  // NOTE: A header can be added to a comment by prepending the comment with header text ending with a colon.
+  const prefix = 'Special Instructions: ';
+  const instructions = appointment.comment
+    ? prefix.concat(appointment.comment)
+    : prefix;
+  const practitionerName = appointment.participant?.find(res =>
+    res.actor.reference.startsWith('Practitioner'),
+  )?.actor.display;
+
+  return (
+    <div>
+      <div className="vads-u-display--block vads-u-padding-y--2p5">
+        ‹ <Link to="/">Manage appointments</Link>
+      </div>
+
+      <h1>
+        <AppointmentDateTime
+          appointmentDate={moment.parseZone(appointment.start)}
+          timezone={appointment.vaos.timeZone}
+          facilityId={getVARFacilityId(appointment)}
+        />
+      </h1>
+
+      <div className="vads-u-font-size--sm vads-u-font-family--sans">
+        <span>
+          <strong>{header}</strong>
+        </span>
+      </div>
+
+      {!!practitionerName ||
+        (location.name && (
+          <>
+            {practitionerName || location.name}
+            <br />
+          </>
+        ))}
+      <FacilityAddress
+        facility={location}
+        showDirectionsLink={!!location.address}
+        isHomepageRefresh
+        isCommunityCare
+      />
+
+      <div className="vads-u-margin-top--3 vaos-appts__block-label">
+        <AppointmentInstructions
```

Comment:
I think I'll make a separate ticket to refactor `AppointmentInstructions`. It looks like this component was used for just VA appointments before and we had a separate CC one, but now it's used for both on the refresh page. I'm not sure we're gaining much vs just putting the header and instructions in this component.